conda-forge.yml

```diff
@@ -1 +1,7 @@
+build_platform: {osx_arm64: osx_64}
 conda_forge_output_validation: true
+# Feb 28, 2021, hmaarrfk
+# We use azure since this is a very small program
+# and we are trying to avoid congestion on drone and travis
+provider: {linux_aarch64: azure, linux_ppc64le: azure}
+test_on_native_only: true
```

Comment:
Is this a file that should be manually edited? I've not seen this before.

Reply:
The migrator would typically modify it for you. But since it is a simple program i figured I would do it manually. 

Test only on native ensures that when we cross compile (on osx) we dontattempt to test the code.

Reply:
ok